### DISCUSSION TITLE
ABC Owned Station News Stream

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#179] [Show and tell]
#EXTINF:0,Dump-WLS-TV-NEWS
https://api.abcotvs.com/v3/wls/m3u8/url?id=567113&key=vuemix
#EXTINF:0,Dump-KTRK-TV-News
https://api.abcotvs.com/v3/ktrk/m3u8/url?id=514944&key=yuemix
#EXTINF:0,Dump-KFSN-News
https://api.abcotvs.com/v3/kfsn/m3u8/url?id=432725&key=yuemix
#EXTINF:0,Dump-WTVD-TV-News
https://api.abcotvs.com/v3/wtvd/m3u8/url?id=563661&key=yuemix

-Will Add in Mybtv Commit Station-
#EXTINF:0,Dump-KGO-TV~News
https://api.abcotvs.com/v3/kgo/m3u8/url?id=530971&key=yuemix
#EXTINF:0,Dump-WABC-News
https://api.abcotvs.com/v3/wabc/m3u8/url?id=503123&key=yuemix
#EXTINF:0,Dump-WPVI-News
https://api.abcotvs.com/v3/wpvi/m3u8/url?id=531099&key=yuemix

-Already in US Playlist-
#EXTINF:0,Dump-KABC-News
https://api.abcotvs.com/v3/kabc/m3u8/url?id=540751&key=yuemix

Link Will Redirect to Each Station's m3u8 Stream
Dumped From Each Station's News App. 
P.S. Only Local News. and Stream Become Laggy During Commercial Break.

**Comment by what-a** [comment]
All are working for me in VLC and IPTVnator on both Windows10 and Linux.  👍
